sdk/src/main/java/com/mapbox/maps/CredentialsManager.kt

```diff
@@ -0,0 +1,43 @@
+package com.mapbox.maps
+
+import android.content.Context
+
+/**
+ * Convenience class that holds MapboxMaps related token.
```

Comment:
Can we use a reference to `MapboxMap` here instead of MapboxMaps?

Reply:
And I guess as per our infrastructure it is more connected with `MapView` and not `MapboxMap`